### ISSUE TITLE
Warning in Julia 1.7

### ISSUE THREAD

**Issue by robertdj** [JuliaInterop/RCall.jl#438]
In Julia 1.7 I get this warning when using RCall:

```
 Warning: use values(kwargs) and keys(kwargs) instead of kwargs.data and kwargs.itr
│   caller = iterate(x::Base.Pairs{Int64, ListSxp, Base.OneTo{Int64}, Ptr{ListSxp}}) at methods.jl:232
└ @ RCall ~/.julia/packages/RCall/iMDW2/src/methods.jl:232  
```

Applying the suggested changes removes the warning. I'd be happy to make a PR.

**Comment by lbilli** [comment]
See #426

**Comment by simonbyrne** [comment]
Fixed by #426
